src/main/java/service/GetFruitFromFile.java

```diff
@@ -0,0 +1,7 @@
+package service;
+
+import model.Fruit;
+
+public interface GetFruitFromFile {
```

Comment:
This is not a usual naming for java interfaces. A more conventional name for your interface might be FruitFileReader or FruitFromFileProvider.